src/kakarot/accounts/uninitialized_account.cairo

```diff
@@ -0,0 +1,43 @@
+// SPDX-License-Identifier: MIT
+
+%lang starknet
+
+// Starkware dependencies
+from openzeppelin.access.ownable.library import Ownable
+from starkware.cairo.common.cairo_builtins import HashBuiltin, BitwiseBuiltin, SignatureBuiltin
+from starkware.cairo.common.uint256 import Uint256
+
+// Local dependencies
+from kakarot.accounts.library import GenericAccount, Account_kakarot_address, Account_evm_address
+from starkware.starknet.common.syscalls import get_tx_info, get_caller_address, replace_class
+from starkware.cairo.common.math import assert_le, unsigned_div_rem
+from starkware.cairo.common.alloc import alloc
+
+@constructor
+func constructor{
+    syscall_ptr: felt*, pedersen_ptr: HashBuiltin*, range_check_ptr, bitwise_ptr: BitwiseBuiltin*
```

Comment:
I don't think that we need the bitwise here

Reply:
I feel like we could completely empty the constructor because we deploy+initialize in the same function. Like that the mapping would be completely future proof, with the Starknet address solely depending on a transparent class

Reply:
Keeping the constructor arguments for now. It makes sure that accounts are bound to a specific deployment of Kakarot, and that external users cannot wrongly deploy + initialize accounts.